stirling-pdf/src/main/resources/messages_zh_TW.properties

```diff
@@ -135,598 +135,640 @@ lang.vie=Vietnamese
+compare.tags=區分,對比,變化,分析
 
-home.certSign.title=??????
-home.certSign.desc=????/???PEM/P12??? PDF
-certSign.tags=??,PEM,P12,??,??
+home.certSign.title=使用憑證簽章
+home.certSign.desc=使用憑證/金鑰（PEM/P12）簽章 PDF
+certSign.tags=驗證,PEM,P12,官方,加密
 
-home.removeCertSign.title=????
-home.removeCertSign.desc=? PDF ????
-removeCertSign.tags=??,PEM,P12,??,??
+home.removeCertSign.title=移除簽章
+home.removeCertSign.desc=從 PDF 移除簽章
+removeCertSign.tags=驗證,PEM,P12,官方,解密
 
-home.pageLayout.title=???????
-home.pageLayout.desc=? PDF ??????????????
-pageLayout.tags=??,??,????,??
+home.pageLayout.title=多頁面版面配置
+home.pageLayout.desc=將 PDF 檔案的多個頁面合併到單一頁面
+pageLayout.tags=合併,複合,單一檢視,組織
 
-home.scalePages.title=??????/??
-home.scalePages.desc=???????????/???
-scalePages.tags=????,??,??,??
+home.scalePages.title=調整頁面大小/比例
+home.scalePages.desc=修改頁面及其內容的大小/比例。
+scalePages.tags=調整大小,修改,尺寸,適應
 
-home.pipeline.title=??????
-home.pipeline.desc=?????????? PDF ???????
-pipeline.tags=???,??,???,????
+home.pipeline.title=管道（進階）
+home.pipeline.desc=透過定義管道指令碼在 PDF 上執行多個操作
+pipeline.tags=自動化,序列,指令碼,批次處理
 
-home.add-page-numbers.title=????
-home.add-page-numbers.desc=????????????
-add-page-numbers.tags=??,??,??,??
+home.add-page-numbers.title=新增頁碼
+home.add-page-numbers.desc=在文件的設定位置新增頁碼
+add-page-numbers.tags=分頁,標籤,組織,索引
 
-home.auto-rename.title=?????? PDF ??
-home.auto-rename.desc=??????????????? PDF ??
-auto-rename.tags=????,????,??,????
+home.auto-rename.title=自動重新命名 PDF 檔案
+home.auto-rename.desc=根據其偵測到的標頭自動重新命名 PDF 檔案
+auto-rename.tags=自動偵測,基於標頭,組織,重新標籤
 
-home.adjust-contrast.title=????/???
-home.adjust-contrast.desc=?? PDF ???????????
-adjust-contrast.tags=????,??,??,??
+home.adjust-contrast.title=調整顏色/對比度
+home.adjust-contrast.desc=調整 PDF 的對比度、飽和度和亮度
+adjust-contrast.tags=色彩校正,調整,修改,增強
 
-home.crop.title=?? PDF
-home.crop.desc=?? PDF ?????????????
-crop.tags=??,??,??,??
+home.crop.title=裁剪 PDF
+home.crop.desc=裁剪 PDF 以減少其大小（保持文字！）
+crop.tags=修剪,縮小,編輯,形狀
 
-home.autoSplitPDF.title=??????
-home.autoSplitPDF.desc=??????? PDF???????????? QR Code
-autoSplitPDF.tags=?? QR Code,??,????,??
+home.autoSplitPDF.title=自動分割頁面
+home.autoSplitPDF.desc=自動分割掃描的 PDF，使用實體掃描頁面分割器 QR Code
+autoSplitPDF.tags=基於 QR Code,分離,掃描區段,組織
 
-home.sanitizePdf.title=??
-home.sanitizePdf.desc=? PDF ?????????????
-sanitizePdf.tags=??,??,??,????
+home.sanitizePdf.title=清理
+home.sanitizePdf.desc=從 PDF 檔案中移除指令碼和其他元素
+sanitizePdf.tags=清理,安全,無害,移除威脅
 
-home.URLToPDF.title=??/??? PDF
-home.URLToPDF.desc=??? http(s) ????? PDF
-URLToPDF.tags=????,????,?????,??
+home.URLToPDF.title=網址/網站轉 PDF
+home.URLToPDF.desc=將任何 http(s) 網址轉換為 PDF
+URLToPDF.tags=網頁捕獲,儲存頁面,網頁轉文件,存檔
 
-home.HTMLToPDF.title=HTML ? PDF
-home.HTMLToPDF.desc=??? HTML ????????? PDF
-HTMLToPDF.tags=??,????,??,??
+home.HTMLToPDF.title=HTML 轉 PDF
+home.HTMLToPDF.desc=將任何 HTML 檔案或壓縮檔轉換為 PDF
+HTMLToPDF.tags=標記,網頁內容,轉換,轉檔
 
 #eml-to-pdf
 home.EMLToPDF.title=Email to PDF
-home.EMLToPDF.desc=Converts email (EML) files to PDF format including headers, body, and inline images
```

Comment:
Looking at these specific lines, there are clearly no encoding issues here, these are standard English text and basic tags. Why were my Traditional Chinese translations for these exact same keys taken from my working-in-progress PR and incorporated here? This has nothing to do with fixing encoding problems.